cogs/assistance-cmds/tinfoil.switch.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Why you shouldn't use tinfoil
+help-desc: Why you shouldn't use tinfoil
+color: 009AC7
+---
+**We don't recommend the usage of tinfoil for the following reason:**
```

Comment:
Nitpick typo, you have more than one reason so it should be in plural "following reasons"